pkg/kudoctl/packages/resolver/resolver.go

```diff
@@ -9,12 +9,6 @@ import (
 	"github.com/kudobuilder/kudo/pkg/kudoctl/util/repo"
 )
 
-// Resolver will try to resolve a given package name to either local tarball, folder, remote url or
```

Comment:
It's not used (but probably should?) I'm not sure. In any case, this seems unrelated to current PR.